core/translate/insert.rs

```diff
@@ -597,8 +656,7 @@ fn populate_column_registers(
             // When inserting a single row, SQLite writes the value provided for the rowid alias column (INTEGER PRIMARY KEY)
             // directly into the rowid register and writes a NULL into the rowid alias column. Not sure why this only happens
             // in the single row case, but let's copy it.
-            let write_directly_to_rowid_reg =
-                mapping.column.is_rowid_alias && !inserting_multiple_rows;
+            let write_directly_to_rowid_reg = mapping.column.is_rowid_alias;
```

Comment:
If you change this then please also change the comment above and ensure it works as intended